src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const invertRobot = {};
+
+  for (const key in robot) {
+    if (invertRobot.hasOwnProperty(robot[key])) {
+      return null;
+    } else {
+      invertRobot[robot[key]] = key;
```

Comment:
I'm not sure, that it's necessary, but you can create a variable 'robot[key]' and use it

Reply:
I agree, it can make sense